app/decorators/application_decorator.rb

```diff
@@ -1,8 +1,13 @@
-class ApplicationDecorator < Draper::Decorator
-  # Define methods for all decorated objects.
-  # Helpers are accessed through `helpers` (aka `h`). For example:
-  #
-  #   def percent_amount
-  #     h.number_to_percentage object.amount, precision: 2
-  #   end
+class ApplicationDecorator
+  delegate_missing_to :@object
```

Comment:
[This article](https://blog.bigbinary.com/2017/05/30/rails-5-1-adds-delegate-missing-to.html) has the following explanation:

> Why not SimpleDelegator
> We at BigBinary have used SimpleDelegator. However one issue with this is that statically we do not know to what object the calls are getting delegated to since at run time the delegator could be anything.

> Why not delegate method
> Delegate method works. However here we need to white list all the methods and in some cases the list can get really long

Here you can also find DHH comments about `SimpleDelegator`: https://github.com/rails/rails/issues/23824#issuecomment-221171899

Reply:
I always preferred the explicit whitelisting.  Makes it much easier to make your decorated objects "dumb"  and avoid things like SQL queries triggered from views etc. Anyway, that ship has sailed for Rails, thanks for the context.